test/e2e/kubernetes/kubernetes_test.go

```diff
@@ -2008,6 +2024,105 @@ var _ = Describe("Azure Container Cluster using the Kubernetes Orchestrator", fu
 				Skip("No windows agent was provisioned for this Cluster Definition")
 			}
 		})
+		It("should expect containers to be recreated after node restart", func() {
+			if eng.HasWindowsAgents() {
+				if eng.ExpandedDefinition.Properties.HasAvailabilityZones() {
```

Comment:
Was this supposed to check for AvailablitySets (agentPoolProfile.availabilityProfile == AvailabilitySet) instead of AvailabilityZones (based on other comments)?

Reply:
Ah yes, I got confused. Will fix